### PR TITLE
Feature/aa 148 save additional accommodation info in booking record

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added AccommodationInfo to Booking model. Added property Photo to AccommodationInfo model. These photo are saved in the BookingAvailabilityInfo at stage booking evaluation and then added to the booking when it is created.

### CODE REVIEW

File: HappyTravel.Edo.Data/Bookings/Booking.cs [hunk 1]
```diff
@@ -45,6 +45,7 @@ public class Booking : IEntity
         
         public string AccommodationId { get; set; }
         public string AccommodationName { get; set; }
+        public AccommodationInfo AccommodationInfo { get; set; }
```
Comment (kirillta) [inline HappyTravel.Edo.Data/Bookings/Booking.cs:48]: Why we have an ID, a name, info, and a location in four difference objects? All of them supposed to be parts of `AccommodationInfo`

Comment (mingazhev) [reply HappyTravel.Edo.Data/Bookings/Booking.cs:48]: Ok, let's have all this in one place. But in a different PR because migration can be complex

**Review by mingazhev — COMMENTED** [review]
Could not find much differences in 2 PRs, are they related?

File: HappyTravel.Edo.Data/Bookings/Booking.cs (hunk 1 above)
Comment (AlexeyShok) [reply HappyTravel.Edo.Data/Bookings/Booking.cs:48]: Ok

**Review by mingazhev — APPROVED** [review]

**Pull request merged**